file/validate.go

```diff
@@ -29,5 +82,16 @@ func validate(content []byte) error {
 		err := fmt.Errorf(desc.String())
 		errs.Errors = append(errs.Errors, err)
 	}
+
+	var kongdefaults *KongDefaults
+	if err := json.Unmarshal(content, kongdefaults); err != nil {
```

Comment:
This won't work. The code should unmarshal this into [Content](https://github.com/Kong/deck/blob/53b3f95b3a991161535a865515dc5f28f6c2b249/file/types.go#L599) struct.

Reply:
we need validate and error out each item within the struct, right. If so, we need pass in the specific DS as a parameter if I understand correctly.